astroquery/mpc/core.py

```diff
@@ -1184,7 +1184,11 @@ def _parse_result(self, result, **kwargs):
 
             if len(src) == 0:
                 raise RuntimeError(('No data queried. Are the target '
```

Comment:
Not directly related to the issue, but `astroquery` defines a more appropriate error type:
https://github.com/astropy/astroquery/blob/f2d0bd23e0155da813cca47427a0102806cc5a30/astroquery/exceptions.py#L109

Reply:
I agree, RuntimeError is not the right one.  EmptyResponseError: Astroquery error class to be raised when the query returns an empty result.  For this error the result is an empty list, i.e., `[]`.  If that is OK, then I'd like to use EmptyResponseError.

Reply:
yes, I think that's appropriate (my reading is that ` EmptyResponseError` should be raised when query results are expected but empty is what's returned (as opposed to legit empty response when e.g. there are no matches with the query parameters, but because there are no observations of the given region etc., that case e.g. an empty table can be returned)

Reply:
Are we saying, then, that we want to have `EmptyResponseError` instead of `RuntimeError` before merging this PR?  (I'm asking mostly to bump this PR, which otherwise looks useful)

Reply:
It would be good to change the error type, but because these lines of code were not introduced in this pull request then I don't think this should be a requirement for merging.